src/main/java/com/toofifty/easyblastfurnace/utils/SessionStatistics.java

```diff
@@ -31,6 +33,11 @@
     @Getter
     private int staminaDoses = 0;
 
+    private Instant timeStarted;
+
+    @Getter
+    private long barsPerHour = 0;
```

Comment:
`long`? How many bars do you think people can make per hour? 😂

Reply:
Good point lol